articles/quickstart/backend/golang/interactive.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Add authorization to a Go API
+description: This tutorial demonstrates how to add authorization to a Go API.
+topics:
+  - quickstart
+  - backend
+  - golang
+interactive: true
+github:
+  path: 01-Authorization-RS256
+contentType: tutorial
+useCase: quickstart
+files:
+  - files/jwt
+  - files/main
+---
+
+<!-- markdownlint-disable MD041 MD025 -->
+
+# Add Authorization to a Go application.
+Auth0 allows you to quickly add authentication and gain access to user profile information in your application. This guide demonstrates how to integrate Auth0 with any new or existing ASP.NET Owin Web API application using the [go-jwt-middleware](https://github.com/auth0/go-jwt-middleware) package.
```

Comment:
> any new or existing ASP.NET Owin Web API application

Should this refer to a Go app as opposed to a ASP.NET Owin app?

Reply:
> Auth0 allows you to quickly add authentication

Should this be "authentication" or "authorization"? (Title refers to authorization but then we're saying "add authentication").

Reply:
👍 copy paste error